testsuite/integration/clustering/src/test/java/org/jboss/as/test/clustering/cluster/ejb/xpc/bean/StatefulBean.java

```diff
@@ -166,9 +168,12 @@ private void logStats(String methodName) {
         session.getSessionFactory().getStatistics().setStatisticsEnabled(true);
         session.getSessionFactory().getStatistics().logSummary();
         String[] entityRegionNames = session.getSessionFactory().getStatistics().getSecondLevelCacheRegionNames();
-        for (String name : entityRegionNames) {
+        String legacyPrefix = session.getSessionFactory().getSessionFactoryOptions().getCacheRegionPrefix() + ".";
+        for (String qualifiedName : entityRegionNames) {
+            log.trace("qualified cache entity region name = " + qualifiedName);
+            String name = qualifiedName.startsWith(legacyPrefix) ? qualifiedName.substring(legacyPrefix.length()) : qualifiedName;
```

Comment:
@scottmarlow I did this startsWith check to make the test robust, i.e. it works when Hibernate 5.3 getSecondLevelCacheRegionNames includes the prefix, but also works if a later version does not.

Reply:
:+1: